CHANGELOG.md

```diff
@@ -2,14 +2,24 @@
 
 Full documentation for hipSOLVER is available at [hipsolver.readthedocs.io](https://hipsolver.readthedocs.io/en/latest/).
 
-## [(unreleased) hipSOLVER for ROCm 4.5.0]
+
+## [(Unreleased) hipSOLVER for ROCm 4.5.0]
```

Comment:
Same as with rocSOLVER:
- remove the "for ROCm 4.5.0", and
- change Optimizations -> Optimized